File: Example Code/02 Branches/03 If Else If/Calculator.cpp
#include <iostream>
using namespace std;

int main()
{
    int a, b;
    cout << "Number 1: ";
    cin >> a;
    cout << "Number 2: ";
    cin >> b;

    cout << endl << "Type of operation?" << endl;
    cout << "1. Add" << endl;
    cout << "2. Subtract" << endl;
    cout << "3. Multiply" << endl;
    cout << "4. Divide" << endl;

    int choice;
    cin >> choice;

    if ( choice == 1 )
    {
        int result = a + b;
        cout << a << " + " << b << " = " << result << endl;
    }
    else if ( choice == 2 )
    {
        int result = a - b;
        cout << a << " - " << b << " = " << result << endl;
    }
    else if ( choice == 3 )
    {
        int result = a * b;
        cout << a << " * " << b << " = " << result << endl;
    }
    else if ( choice == 4 )
    {
        int result = a / b;
        cout << a << " / " << b << " = " << result << endl;
    }
    else
    {
        cout << "Invalid option" << endl;
    }

    return 0;
}

File: Example Code/02 Branches/03 If Else If/Calculator.py
a = float( input( "Enter number 1: " ) )
b = float( input( "Enter number 2: " ) )

print( "\nWhat kind of operation?" )
print( "1. Add \t 2. Subtract \t 3. Multiply \t 4. Divide" )

choice = input( ">> " )

if ( choice == 1 ):
	result = a + b
	print( str( a ) + " + " + str( b ) + " = " + str( result ) )
	
elif ( choice == 2 ):
	result = a - b
	print( str( a ) + " - " + str( b ) + " = " + str( result ) )
	
elif ( choice == 3 ):
	result = a * b
	print( str( a ) + " * " + str( b ) + " = " + str( result ) )
	
elif ( choice == 4 ):
	result = a / b
	print( str( a ) + " / " + str( b ) + " = " + str( result ) )

else:
	print( "Invalid operation" )

